ffmpeg_plugin/0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch

```diff
@@ -537,7 +539,7 @@ index 0000000..b2a6ac4
 +      OFFSET(enc_mode), AV_OPT_TYPE_INT, { .i64 = 9 }, 0, 12, VE },
 +
 +    { "profile", "Profile setting, Main Still Picture Profile not supported", OFFSET(profile),
-+      AV_OPT_TYPE_INT, { .i64 = FF_PROFILE_HEVC_MAIN_10 }, FF_PROFILE_HEVC_MAIN, FF_PROFILE_HEVC_REXT, VE, "profile"},
++      AV_OPT_TYPE_INT, { .i64 = FF_PROFILE_HEVC_MAIN }, FF_PROFILE_HEVC_MAIN, FF_PROFILE_HEVC_REXT, VE, "profile"},
```

Comment:
should it be: { .i64 = FF_PROFILE_HEVC_MAIN }, **FF_PROFILE_HEVC_MAIN_10**, ?

Reply:
The min value is FF_PROFILE_HEVC_MAIN, but not FF_PROFILE_HEVC_MAIN_10.